src/components/myreserve/myreserve.js

```diff
@@ -1,6 +1,99 @@
+import { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getMyReservations, cancelReservation } from '../../redux/reservations/reservetions';
+import avatarBike from '../../icons/bike-icon.png';
+import './myreserve.css';
+
 function ReservePage() {
```

Comment:
- [ ] In the `reserve, and motorcycles` sections, the images won't render.  

![image](https://user-images.githubusercontent.com/57421823/193008109-857cc045-7273-4707-acc4-15b0feba7abe.png)

![image](https://user-images.githubusercontent.com/57421823/193008157-f340319d-29bf-4d70-9f1b-70ebbdfaa843.png)

Please, kindly correct this issue so your app works as intended 💪

Reply:
- The image at this point doesn't work. Please fix this important bug.
![image](https://user-images.githubusercontent.com/20563252/193062576-a0864379-a3ef-47c8-9893-0c9dfda7e7e8.png)